Library/Formula/tdb.rb

```diff
@@ -0,0 +1,68 @@
+class Tdb < Formula
+  homepage "http://sourceforge.net/projects/tdb/"
+  url "https://downloads.sourceforge.net/project/tdb/tdb/1.0.6/tdb-1.0.6.tar.gz"
+  sha1 "d1876522f1b8ffa8cf844a1f6605e0c32d387a7a"
+
+  depends_on "gdbm" => :build
+
+  # older versions of the compiler allowed you to have a printf that
+  # spaned multiple lines, now that's no longer the case. There is a
+  # patch submitted upstream for this but it doesn't seem like it will
+  # ever be applied: http://sourceforge.net/p/tdb/patches/6/ as it
+  # dates back to 2003, and still hasn't been.
+  patch :DATA
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--mandir=#{man}",
+                          "--host=6x86"
```

Comment:
Was the only way I could get the configure script to not error out.  The auto host detection doesn't understand x86_64 -- I'll see if there's a better way I can deal with this.  Thanks @mikemcquaid